libs/libuwsc/Makefile

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (C) 2014-2017 OpenWrt.org
```

Comment:
Though I am not a lawyer, it's very likely that you should use year 2018 and your own name in the copyright header.